src/Workspaces/Core/Portable/Workspace/Solution/AssetHint.cs

```diff
@@ -27,6 +27,8 @@ namespace Microsoft.CodeAnalysis.Serialization;
     /// </summary>
     public static readonly AssetPath SolutionAndTopLevelProjectsOnly = new(AssetPathKind.Solution | AssetPathKind.TopLevelProjects);
 
+    public static readonly AssetPath ProjectOnly = new(AssetPathKind.Projects);
```

Comment:
>AssetPath [](http://example.com/codeflow?start=27&length=9)

nit: might be nice to rename this file to match the recent name change

Reply:
will do.